3rd-party/CMakeLists.txt

```diff
@@ -1,3 +1,3 @@
 add_subdirectory(spdlog)
 add_subdirectory(json)
-add_subdirectory(libpqxx)
+#add_subdirectory(libpqxx)
```

Comment:
Why commented? Should it be removed or ?

Reply:
It shouldn't, I just don't have libpqq installed. This will be added back before merge.